addons/purchase/views/purchase_views.xml

```diff
@@ -367,6 +368,13 @@
                                                              </span>
                                                          </div>
                                                      </div>
+                                                     <div class="row" t-if="record.discount.raw_value">
+                                                         <div class="col-12 text-muted">
+                                                            <span>
+                                                                Discount: <t t-esc="record.discount.value + '%'"/>
```

Comment:
same comment about the %, you should be able to rearrange it to not be in the `t-esc`